### PR TITLE
Virtual Wan P2S MultiPool feature changes

### PR DESCRIPTION
**Pull request by Nilambari**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1127 
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network.Test/ScenarioTests/CortexTests.ps1 [hunk 1]
```diff
@@ -746,17 +760,17 @@ function Test-CortexExpressRouteCRUD
 		Assert-AreEqual $vpnServerConfig1.Id $P2SVpnGateway.VpnServerConfiguration.Id
 		Assert-AreEqual "Succeeded" $P2SVpnGateway.ProvisioningState
 		Assert-AreEqual 1 @($P2SVpnGateway.CustomDnsServers).Count
-        	Assert-AreEqual "7.7.7.7" $P2SVpnGateway.CustomDnsServers[0]
+        Assert-AreEqual "7.7.7.7" $P2SVpnGateway.CustomDnsServers[0]
```
Comment (BethanyZhou) [inline src/Network/Network.Test/ScenarioTests/CortexTests.ps1:763]:
```suggestion
        	Assert-AreEqual "7.7.7.7" $P2SVpnGateway.CustomDnsServers[0]
```

File: src/Network/Network.Test/ScenarioTests/CortexTests.ps1 [hunk 2]
```diff
@@ -746,17 +760,17 @@ function Test-CortexExpressRouteCRUD
...
 		# Reset/Reboot the P2SVpnGateway using Reset-AzP2sVpnGateway
-        	$job = Reset-AzP2sVpnGateway -P2SVpnGateway $P2SVpnGateway -AsJob
-	    	$job | Wait-Job
-	    	$actual = $job | Receive-Job
-		
-		$P2SVpnGateway = Get-AzP2sVpnGateway -ResourceGroupName $rgName -Name $P2SvpnGatewayName
-		Assert-AreEqual "Succeeded" $P2SVpnGateway.ProvisioningState
+        $job = Reset-AzP2sVpnGateway -P2SVpnGateway $P2SVpnGateway -AsJob
+	    $job | Wait-Job
+	    $actual = $job | Receive-Job
```
Comment (BethanyZhou) [inline src/Network/Network.Test/ScenarioTests/CortexTests.ps1:773]: Please correct indents.

File: src/Network/Network/Az.Network.psd1 [hunk 3]
```diff
@@ -436,10 +436,12 @@ CmdletsToExport = 'Add-AzApplicationGatewayAuthenticationCertificate',
                'Get-AzVpnConnection', 'Update-AzVpnConnection', 
                'Remove-AzVpnConnection', 'New-AzVirtualHubVnetConnection', 
                'Get-AzVirtualHubVnetConnection', 
-               'Remove-AzVirtualHubVnetConnection', 
-               'Update-AzVirtualHubVnetConnection', 'Get-AzVpnServerConfiguration', 
-               'New-AzVpnServerConfiguration', 'Remove-AzVpnServerConfiguration', 
-               'Update-AzVpnServerConfiguration', 'Get-AzP2sVpnGateway', 
+               'Remove-AzVirtualHubVnetConnection', 'Update-AzVirtualHubVnetConnection',
```
Comment (BethanyZhou) [inline src/Network/Network/Az.Network.psd1:439]: Please correct indents.

**Comment by BethanyZhou** [comment]
CI failed: The type or namespace name 'IConfigurationPolicyGroupsOperations' could not be found (are you missing a using directive or an assembly reference?)  Please have a look.

**Comment by Nilambari** [comment]
@isra-fel  Incorporate review comment. Please take a look.

**Comment by Nilambari** [comment]
> CI failed: The type or namespace name 'IConfigurationPolicyGroupsOperations' could not be found (are you missing a using directive or an assembly reference?) Please have a look.

Waiting for latest network SDK generation.

**Comment by isra-fel** [comment]
Generally looks good. Please regenerate help documents and resolve the conflicts.

**Comment by Nilambari** [comment]
> Generally looks good. Please regenerate help documents and resolve the conflicts.

Done. @isra-fel Please take a look. Thanks!

**Review by isra-fel — CHANGES_REQUESTED** [review]
This file `src/Network/Network/help/Az.Network.md` is supposed to be updated when introducing new cmdlets.
Please make sure to use `-RefreshModulePage` when regenerating helps, see https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

**Comment by nilambarideshpande** [comment]
> This file `src/Network/Network/help/Az.Network.md` is supposed to be updated when introducing new cmdlets.
> Please make sure to use `-RefreshModulePage` when regenerating helps, see https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

Let me update it. Thanks.

**Comment by Nilambari** [comment]
> Let me update it. Thanks.

Incorporated review comments & included latest test recording as well. @isra-fel  Please take a look and help approve this PR. Thanks!

**Review by isra-fel — APPROVED** [review]

**Pull request merged**